### PR TITLE
API 37228 mpi fix headers first & last name check

### PR DESCRIPTION
**Pull request by rockwellwindsor-va**
## Summary
* Updates header check to also verify first name and last name are not empty string
* Adds one additional test for the scenario

## Related issue(s)
[API-37228](https://jira.devops.va.gov/browse/API-37228)

## Testing done
- [x] *New code is covered by unit tests*

## What areas of the site does it impact?
	modified:   modules/claims_api/app/controllers/claims_api/v1/application_controller.rb
	modified:   modules/claims_api/spec/requests/v1/power_of_attorney_request_spec.rb

## Acceptance criteria
- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: modules/claims_api/app/controllers/claims_api/v1/application_controller.rb [hunk 1]
```diff
@@ -193,17 +193,18 @@ def edipi_check
       end
 
       def validate_header_values_format
-        if header('X-VA-Birth-Date').blank?
+        errors = []
+        errors << 'birth date' if header('X-VA-Birth-Date').blank?
+        errors << 'first name' if header('X-VA-First-Name').blank?
+        errors << 'last name' if header('X-VA-Last-Name').blank?
```
Comment (FonzMP) [inline modules/claims_api/app/controllers/claims_api/v1/application_controller.rb:199]: more nitpick but should we include the header value as it's directly inline with what they're "missing" 

so something like 
```
errors << 'X-VA-Birth-Date' if header('X-VA-Birth-Date').blank?
```

**Review by FonzMP — DISMISSED** [review]
:shipit:

File: modules/claims_api/app/controllers/claims_api/v1/application_controller.rb (hunk 1 above)
Comment (rockwellwindsor-va) [reply modules/claims_api/app/controllers/claims_api/v1/application_controller.rb:199]: Ya, I was not sure what they saw from their end, but that is an easy enough change to make here I will update it

**Review by FonzMP — APPROVED** [review]
:shipit:

**Pull request merged**